lib/ClangImporter/ClangImporter.cpp

```diff
@@ -4024,22 +4036,47 @@ bool ClangImporter::Implementation::forEachLookupTable(
 bool ClangImporter::Implementation::lookupValue(SwiftLookupTable &table,
                                                 DeclName name,
                                                 VisibleDeclConsumer &consumer) {
+
   auto &clangCtx = getClangASTContext();
   auto clangTU = clangCtx.getTranslationUnitDecl();
 
   bool declFound = false;
-
   // For operators we have to look up static member functions in addition to the
   // top-level function lookup below.
+  auto declBaseName = (SwiftContext.LangOpts.EnableCXXInterop && name.isOperator())
+          ? DeclBaseName(SwiftContext.getIdentifier("__operator" + getOperatorNameForToken(std::string{name.getBaseName().getIdentifier()})))
+          : name.getBaseName();
+
   if (name.isOperator()) {
-    for (auto entry : table.lookupMemberOperators(name.getBaseName())) {
-      if (isVisibleClangEntry(entry)) {
-        if (auto decl = dyn_cast_or_null<ValueDecl>(
-                importDeclReal(entry->getMostRecentDecl(), CurrentVersion))) {
-          consumer.foundDecl(decl, DeclVisibilityKind::VisibleAtTopLevel);
-          declFound = true;
+    for (auto entry : table.lookupMemberOperators(declBaseName)) {
```

Comment:
It seems like those two loops are exact the same code(just over different lists), could this be abstracted into a function/lambda in some way?

Reply:
Yes I think so! Just gotta rebase and fix some tests first.